design/docs-reorg.md

```diff
@@ -0,0 +1,73 @@
+# Documentation Reorganization
+
+This document proposes a new outline for the Velero documentation that is intended to be easier for both users and contributors to interact with.
+
+## Goals
+
+- Agree on the desired outline (sections and pages) for Velero documentation.
+
+## Non Goals
+
+- Write/re-write all new/revised pages (individual page writes/rewrites to be tracked as individual issues).
+
+## Background
+
+The Velero documentation has evolved organically since the project's inception.
+While it contains a lot of useful information, it is not currently organized in a way that is easy to navigate for users or contributors.
+For example, the introductory section jumps straight into a discussion of Velero's architecture, rather than starting with a good overview of its core features.
+Also, the documentation on how to actually *use* Velero is fairly superficial, only covering the most straight-forward use cases.
+Additionally, different types of information are often mixed together (i.e. conceptual overviews along with detailed setup instructions).
+
+This design document proposes a new outline for the documentation that is intended to be more easily navigated by both users and contributors.
+The new outline both reorganizes the existing content, *and* calls for new content where relevant.
+
+## High-Level Design
```

Comment:
Who will be the target audience of the doc that will be created? That would be a good indicator of how we want to organize this.

Reply:
IMHO the main audience is users. There's also a "core contributors" audience and a "plugin developers" audience. I think that covers it but LMK if you think there are others!

Reply:
So this is what I am thinking. We can have like a Quick Start doc that folks would use to kick the tires and get see velero in action. This is where we would cover velero installation, backup store setup and volume snapshotter setup for public clouds. Then there can be advanced user docs that would, maybe, a deep dive into some scenarios.
The plugin developers doc would be a separate doc.